src/main/kotlin/com/terraformation/backend/search/NestedQueryBuilder.kt

```diff
@@ -1142,9 +1148,15 @@ class NestedQueryBuilder(
 
     val primaryKey = rootTable.primaryKey
 
+    // the criteria for a sublist filter will already be in conditions instead of the secondary
+    // table joins since the conditions don't need the table joined
+    val secondaryTablesCriteria = if (isSublistFilter) NoConditionNode() else rootCriterion
```

Comment:
@sgrimm this _seems_ to fix the problem without the need for an alias. What I haven't figured out yet is if `rootCriterian` might have other sublists referenced that would be used in `joinWithSecondaryTables` that are not based on the same table as the current sublist.

Reply:
I _think_ that with the way that sublist filtering works, `rootCriterion` will always only be needed for the `conditions` and thus shouldn't be in the `secondaryTablesCriteria`, but it's possible I'm not thinking through everything. Test for this are in `SearchServiceSublistSearchTest` and include `AndNode` and `OrNode` for these

Reply:
I will definitely confirm, but I believe that this no longer touches the accession search sql since it is only for sublist filtering which is only used by the Matrix View currently.